### DISCUSSION TITLE
More gap on the left and right end of file list column texts

### DISCUSSION THREAD

**Discussion by jdp1024** [doublecmd/doublecmd#2037] [Suggestions]
Currently the gaps between the both ends and the vertical line are too small.

![2024-12-29_205736](https://github.com/user-attachments/assets/8552189d-2ff5-4b31-9e33-6cd4d59977c4)

This is because the CELL_PADDING is a fixed value and never scaled according the screen DPI.

A workaround:

```
procedure DrawOtherCell;
  //------------------------------------------------------
  var
    tw, vTextLeft, ActPadding: Integer;
  begin
    s := AFile.DisplayStrings.Strings[ACol];
    ActPadding := ScaleX(CELL_PADDING,96);

    if gCutTextToColWidth then
      s := FitOtherCellText(s, Canvas, ARect.Width - 2*ActPadding);

    case ColumnsSet.GetColumnAlign(ACol) of

      taRightJustify:
        begin
          tw := Canvas.TextWidth(s);
          vTextLeft := aRect.Right - tw - ActPadding;
          if aCol = ColCount - 1 then
            Dec(vTextLeft, ActPadding);
          Canvas.TextOut(vTextLeft, iTextTop, s);
        end;

      taLeftJustify:
        begin
          Canvas.TextOut(aRect.Left + ActPadding, iTextTop, s);
        end;

      taCenter:
        begin
          tw := Canvas.TextWidth(s);
          Canvas.TextOut((aRect.Left + aRect.Right - tw) div 2, iTextTop, s);
        end;

    end; //of case
  end; //of DrawOtherCell
```

After this, the file list looks more relaxed.

![2024-12-29_210025](https://github.com/user-attachments/assets/f3603005-d9e5-48f7-a23d-931c40f5ef28)

**Comment by alexx2000** [comment]
I guess `DrawIconCell` also should use a scaled `CELL_PADDING`.

**Comment by advice2020** [comment]
I noticed this in the past as well.
I cannot comment on the solution mentioned, but I agree that a little more space in between could help things a little, visually.
For me this occurs/spacing remains the same no matter how much file pane font is increased or decreased (Ctrl+MSW)
